hazzy/hazzy.py

```diff
@@ -107,38 +107,47 @@
 
 # Throw up a dialog with debug info when an error is encountered
 def excepthook(exc_type, exc_value, exc_traceback):
+
+    error_dialog = Dialogs(2)
```

Comment:
hmmmm

the question is, where should the exceptionhook go?

havent worked with this